internal/app/singularity/registry.go

```diff
@@ -115,17 +121,25 @@ func (l *Library) pullAndVerify(ctx context.Context, imgMeta *scs.Image, from, t
 // If so, the image will be copied to the location provided.
 func (l *Library) copyFromCache(hash, name, to string) error {
 	exists, err := l.cache.LibraryImageExists(hash, name)
-	if err != nil {
+	if err == cache.ErrBadChecksum {
+		sylog.Debugf("cache hash does not match, removing old cache...")
```

Comment:
What's the point of two log entries (debug and warning) here? 
If log level is debug, then a warning will be displayed as well, I think this line should be removed.

Reply:
I added it just to have a little more verbosity, but I'll remove it.